stylus/ui-components/button.styl

```diff
@@ -74,32 +74,44 @@ $button--danger
     color             white
 
     &:active
-    &:hover
+    &:not([disabled]):not([aria-disabled=true]):hover
```

Comment:
Every style applied on `:hover` should be applied on `:focus` either (3 times in the PR).